From 861c9f8c1dd0e1f1679a983c79bdf4e9d233efdf Mon Sep 17 00:00:00 2001
From: zhangmeng <zhangmeng.kevin@spacemit.com>
Date: Wed, 28 May 2025 16:40:55 +0800
Subject: [PATCH 32/43] riscv: config: enable CONFIG_RISCV_ISA_ZICBOM for
 spacemit k1

community inclusion
category: feature

--------------------------------

Zicbom extension is needed for non-coherent DMA platform
such as spacemit k1. Ohterwise, there may be some cache
conherent issues.

Signed-off-by: zhangmeng <zhangmeng.kevin@spacemit.com>
---
 arch/riscv/configs/k1_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/riscv/configs/k1_defconfig b/arch/riscv/configs/k1_defconfig
index 1940cec263515..6d6d9caacbc54 100644
--- a/arch/riscv/configs/k1_defconfig
+++ b/arch/riscv/configs/k1_defconfig
@@ -4,6 +4,7 @@
 CONFIG_SOC_SPACEMIT=y
 CONFIG_SOC_SPACEMIT_K1=y
 CONFIG_SOC_SPACEMIT_K1X=y
+CONFIG_RISCV_ISA_ZICBOM=y
 CONFIG_SPACEMIT_K1X_CCU=y
 CONFIG_RESET_K1X_SPACEMIT=y
 CONFIG_PINCTRL_SPACEMIT_K1X=y
-- 
2.51.0

